client/pages/project-view.jsx

```diff
@@ -58,7 +59,7 @@ export default function ProjectView() {
               <p>Zipcode: {project.zipcode}</p>
               <p>Notes: {project.notes}</p>
               <p>Completed: {project.completed === true ? <span>Yes</span> : <span>No</span> } </p>
-              <p>Assigned To: {project.assignedTo === null ? <span>Not Assigned Yet</span> : project.assignedTo}</p>
+              <p>Assigned To: {project.assignedTo === null ? <span>Not Assigned Yet</span> : `${techs.find((tech) => tech.userId === project.assignedTo)?.firstName} ${techs.find((tech) => tech.userId === project.assignedTo)?.lastName}`}</p>
```

Comment:
Since you know `project`, `user`, and `techs` up at line 30, you could calculate `firstName` and `lastName` up there , once, making this code simpler.